app/models/course.rb

```diff
@@ -2,4 +2,13 @@ class Course < ApplicationRecord
 belongs_to :teacher
 has_many :schedules, dependent: :destroy
 has_many :enrollments, dependent: :destroy
+
+validates :name, length: { maximum: 100 }, presence: true
+validates :description, presence: true
+validates :max_num_students, presence: true
+validates :teacher_id, presence: true
```

Comment:
Great job @J2ZROMERO on your project, only thing I noticed that might cause problems in the future is that a user can sign up using the same email multiple times, could you please add validation for the email column of users so that each email can only be used one?